dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/FreemarkerTest.java

```diff
@@ -61,7 +63,7 @@ public static void setUp() throws IOException {
 
   @AfterClass
   public static void cleanUp() throws IOException {
-    // MoreFiles.deleteRecursively(outputDirectory, RecursiveDeleteOption.ALLOW_INSECURE);
+    MoreFiles.deleteRecursively(outputDirectory);
```

Comment:
As per [Javadoc](https://google.github.io/guava/releases/21.0/api/docs/com/google/common/io/RecursiveDeleteOption.html#ALLOW_INSECURE), ALLOW_INSECURE is for symbolic links on (insecure) file systems. 
This test does not use symbolic links.